### PR TITLE
feat: suppress multiple resources in an array

### LINKED ISSUE #1086: feat: suppress the same rules for multiple resources in an array
### Description

Make it possible to suppress the same rules for multiple resources in an array.

### Use Case

When you have multiple similar resources and want to suppress the same rules, it will be nice to be able to suppress all of them without code duplication.

### Proposed Solution

Allowing passing multiple resources/paths in an array when using `addResourceSuppressions` and `addResourceSuppressionsByPath`

```typescript
NagSuppressions.addResourceSuppressions(
  [resource1, resource2],
  [{ id: 'AwsSolutions-EC23', reason: 'lorem ipsum' }]
);
```

```typescript
NagSuppressions.addResourceSuppressionsByPath(
  this,
  ['/path/to/resource1', '/path/to/resource2'],
  [{ id: 'AwsSolutions-EC23', reason: 'lorem ipsum' }]
);
```

### Other information

_No response_

### Acknowledge

- [X] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### PR DESCRIPTION
**Pull request by clueleaf**
Fixes #1086

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,7 +83,34 @@ export class CdkTestStack extends Stack {
 </details>
 
 <details>
-  <summary>Example 2) Child Constructs</summary>
+  <summary>Example 2) Multiple resources</summary>
```
Comment (dontirun) [inline README.md:86]:
```suggestion
  <summary>Example 2) On Multiple Constructs</summary>
```

**Review by dontirun — CHANGES_REQUESTED** [review]
Small documentation change, then we should be good to go!

File: README.md (hunk 1 above)
Comment (clueleaf) [reply README.md]: Thank you!

**Review by dontirun — APPROVED** [review]
Thanks for the contribution!

**Pull request merged**